<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleLogo</class>
 <widget class="QWidget" name="VehicleLogo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}</string>
  </property>
  <widget class="QLabel" name="lblB1">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>23</y>
     <width>195</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
font: 20px, &quot;SimHei&quot;;
color: black;
border: 0px;</string>
   </property>
   <property name="text">
    <string>Mp1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRightHead1">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>23</y>
     <width>28</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
font: 20px, &quot;SimHei&quot;;
color: black;
border: 0px;
border-radius: 14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblB2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>23</y>
     <width>195</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
font: 20px, &quot;SimHei&quot;;
color: black;
border: 0px;</string>
   </property>
   <property name="text">
    <string>Mp2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblA1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>23</y>
     <width>170</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
color: black;
font: 20px, &quot;SimHei&quot;;
border: 0px;</string>
   </property>
   <property name="text">
    <string>Tc1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLeftHead">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>23</y>
     <width>14</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
font: 20px, &quot;SimHei&quot;;
color: black;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblA2">
   <property name="geometry">
    <rect>
     <x>697</x>
     <y>23</y>
     <width>170</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
font: 20px, &quot;SimHei&quot;;
color: black;
border: 0px;</string>
   </property>
   <property name="text">
    <string>Tc2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLeftHead1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>23</y>
     <width>28</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
font: 20px, &quot;SimHei&quot;;
color: black;
border: 0px;
border-radius: 14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRightHead">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>23</y>
     <width>14</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(160, 160, 164);
font: 20px, &quot;SimHei&quot;;
color: black;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pantograph1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>5</y>
     <width>66</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 0px;
background-color: black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pantograph2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>5</y>
     <width>66</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 0px;
background-color: black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tc1Direction">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tc2Direction">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_allClose1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>23</y>
     <width>10</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_allClose2">
   <property name="geometry">
    <rect>
     <x>858</x>
     <y>23</y>
     <width>10</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_HSCB1">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>23</y>
     <width>21</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:12px, &quot;SimHei&quot;;</string>
   </property>
   <property name="text">
    <string>H
S
C
B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_HSCB2">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>23</y>
     <width>21</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font:12px, &quot;SimHei&quot;;</string>
   </property>
   <property name="text">
    <string>H
S
C
B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lblLeftHead1</zorder>
  <zorder>lblB1</zorder>
  <zorder>lblRightHead1</zorder>
  <zorder>lblB2</zorder>
  <zorder>lblA1</zorder>
  <zorder>lblLeftHead</zorder>
  <zorder>lblA2</zorder>
  <zorder>lblRightHead</zorder>
  <zorder>pantograph1</zorder>
  <zorder>pantograph2</zorder>
  <zorder>label_tc1Direction</zorder>
  <zorder>label_tc2Direction</zorder>
  <zorder>label_allClose1</zorder>
  <zorder>label_allClose2</zorder>
  <zorder>label_HSCB1</zorder>
  <zorder>label_HSCB2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
